osu.Game/Online/Chat/MessageFormatter.cs

```diff
@@ -269,10 +269,10 @@ private static MessageFormatterResult format(string toFormat, int startIndex = 0
             handleAdvanced(advanced_link_regex, result, startIndex);
 
             // handle editor times
-            handleMatches(time_regex, "{0}", "osu://edit/{0}", result, startIndex, LinkAction.OpenEditorTimestamp);
+            handleMatches(time_regex, "{0}", $"{OsuGameBase.OSU_PROTOCOL}edit/{0}", result, startIndex, LinkAction.OpenEditorTimestamp);
```

Comment:
This needs more escaping (see test failures):

```suggestion
            handleMatches(time_regex, "{0}", $"{OsuGameBase.OSU_PROTOCOL}edit/{{0}}", result, startIndex, LinkAction.OpenEditorTimestamp);
```